src/main/osd/osd_elements.c

```diff
@@ -1017,6 +1017,23 @@ static void osdElementPower(osdElementParms_t *element)
     tfp_sprintf(element->buff, "%4dW", getAmperage() * getBatteryVoltage() / 10000);
 }
 
+static void osdElementRcChannels(osdElementParms_t *element)
+{
+    const uint8_t xpos = element->elemPosX;
+    const uint8_t ypos = element->elemPosY;
+
+    for (int i = 0; i < OSD_RCCHANNELS_COUNT; i++) {
+        if (osdConfig()->rcChannels[i] >= 0) {
+            int data = constrain(rcData[osdConfig()->rcChannels[i]] - 1000, -1000, 1000);
```

Comment:
Normal channel range is 1000..2000 (PWM_RANGE_MIN .. PWM_RANGE_MAX), with possible overflow on both sides - PWM_PULSE_MIN   .. PWM_PULSE_MAX.

This line will map range to -250 .. 1250, is that desired ?

Reply:
I assumed "full RC range" being 0..2000 from what mikeller mentioned; just push whatever value in this range at the transmitter side and it comes out at the receiver. If this is not the case, the whole scheme must be redesigned.

Reply:
> I assumed "full RC range" being 0..2000 from what mikeller mentioned; just push whatever value in this range at the transmitter side and it comes out at the receiver. If this is not the case, the whole scheme must be redesigned.

I agree with this, it makes the most sense.

Reply:
Sorry, I wasn't clear. My proposal implied scaling up by a factor of 2 to get to a full decimal - alternatives would be to leave unscaled at -500...500, or scale down to -100...100. Not sure what is best.

Reply:
> Sorry, I wasn't clear. My proposal implied scaling up by a factor of 2 to get to a full decimal - alternatives would be to leave unscaled at -500...500, or scale down to -100...100. Not sure what is best.

Full scale (total 2000 values) would probably be best as it has the most resolution, I can't think of a situation where scaled down would be better.  Or perhaps I misunderstand?

Reply:
Full scale in RC is 1000...2000, so 1000 values. If you want to be centered at 0 that gives you -500...500.

Reply:
> Full scale in RC is 1000...2000, so 1000 values. If you want to be centered at 0 that gives you -500...500.

Oh... right.  duuh :)   Positive values would likely be more useful, but I don't think it's all that important.